frontend/awx/resources/inventories/InventoryForm.tsx

```diff
@@ -258,14 +258,14 @@ export function EditInventory() {
   }
 
   const title =
-    inventory.kind === ''
+    inventory?.kind === ''
```

Comment:
Is this necessary? Im not against it, but checks above (isLoaded) should have covered the situations when inventory is null, did you encountered some situations where it skips returning the loading page and threw exception on inventory.kid?

Reply:
No this isn't necessary these are just remnants after rebasing. I will go ahead and remove it